flask_googlemaps/__init__.py

```diff
@@ -98,9 +98,9 @@ def build_marker_dict(self, marker, icon=None):
         return marker_dict
 
     def add_marker(self, lat=None, lng=None, **kwargs):
-        if lat:
+        if lat != None:
```

Comment:
```python
if lat is not None:
    # ...
```
is the Pythonic idiom for testing that a variable is not set to None. This idiom has particular uses in the case of declaring keyword functions with default parameters. is tests identity in Python. Because there is one and only one instance of None present in a running Python script/program, is is the optimal test for this.

Can you please change to `if lat is not None` and `if lng is not None`

Thanks for your collaboration!